### ISSUE TITLE
use with vim-multiple-cursors

### ISSUE THREAD

**Issue by khmelevskii** [907th/vim-auto-save#4]
If i use https://github.com/terryma/vim-multiple-cursors your plugin saved after each changes, but not end multiply editing. How can I fix it?

**Comment by ldong** [comment]
@907th  I have the same issue
Vim version 7.3.429, got it form ubuntu 12.04

```vimrc
" vim-session autosave                                                                                        
let g:auto_save = 1                                                                
let g:session_autoload = 'no'                                                      
let g:session_autosave = 'yes'                                                     
let g:session_autosave_periodic = 3  
```

Any way to support the compatibility for 7.3? Thanks.

**Comment by sfahlberg** [comment]
I get the same issue as @ghost

**Comment by leoroos** [comment]
seems resolved with #23
